app/src/main/res/layout/activity_main.xml

```diff
@@ -7,12 +7,171 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/textView2"
```

Comment:
You should use descriptive names for your IDs so that your code is easier to navigate.